tools/run-dev.py

```diff
@@ -325,18 +325,25 @@ def shutdown_handler(*args: Any, **kwargs: Any) -> None:
     else:
         io_loop.stop()
 
-# log which services/ports will be started
-print("Starting Zulip services on ports: web proxy: {},".format(proxy_port),
-      "Django: {}, Tornado: {}, Thumbor: {}".format(django_port, tornado_port, thumbor_port),
-      end='')
-if options.test:
-    print("")  # no webpack for --test
-else:
-    print(", webpack: {}".format(webpack_port))
+def print_listeners() -> None:
+    print("\nZulip services will listen on ports:")
+    ports = [
+        (proxy_port, CYAN + 'web proxy' + ENDC),
+        (django_port, 'Django'),
+        (tornado_port, 'Tornado'),
+    ]
```

Comment:
I'm a little unhappy about the extra vertical space this consumes.  I really wish Django's `manage.py runserver` let us suppress some of its output, which largely duplicates this notice.  Maybe we should just print the web proxy port, with color?  That's the only one users should care about, and this notice mainly exists to avoid people being confused by:

```
Django version 2.2.12
Tornado server is running at http://127.0.0.1:9993/
Quit the server with CTRL-C.
2020-05-01 17:43:27.153 INFO [:9993] Tornado 9993 loaded 0 event queues in 0.000s
System check identified no issues (1 silenced).
May 01, 2020 - 17:43:27
Django version 2.2.12, using settings 'zproject.settings'
Starting development server at http://127.0.0.1:9992/
Quit the server with CONTROL-C.
```

Reply:
Maybe the right answer is to just clean that underlying debugging output.

Reply:
I actually think it's helpful to know that several ports are in use.  The vertical space doesn't bother me, because it doesn't actually push anything off the screen other than 'Clearing memcached ...', and you have time to read it as the subsequent things are kinda slow.